lib/vmdb/fast_gettext_helper.rb

```diff
@@ -65,7 +65,7 @@ def self.register_locales
                                   :path           => locale_path,
                                   :type           => :po,
                                   :report_warning => false)
-      FastGettext.available_locales = find_available_locales
+      FastGettext.default_available_locales = find_available_locales
```

Comment:
@matthewd @imtayadeway do we need a comment as why we're changing that?  The error puzzled 4 of us... We probably need to warn someone from changing this line again :smile: